Tutorial.md

```diff
@@ -162,6 +162,17 @@ Submit-Renewal -AllAccounts
 
 These are designed to be used in a daily scheduled task. **Make sure to have it run as the same user you're currently logged in as** because the module config is all stored in your local profile. Each day, it will check the existing certs for ones that have reached the renewal window and renew them. It will just ignore the ones that aren't ready yet.
 
+### Updating DNS Plugin Parameters on Renewal
+
+If you are authenticating to your DNS provider using a short-lived access token, you may need to update said token when it comes to renewal. You can do this by specifying the new plugin parameters using the ```-PluginArgs``` parameter. **The full set of plugin arguments must be specified**.
```

Comment:
Thanks for including the update to the tutorial with this. I'd just tweak it slightly to include cases where normal credentials have changes as well.

```suggestion
DNS provider credentials can change over time and some plugins can be used with purposefully short-lived access tokens. In these cases, you can specify the new plugin parameters using the ```-PluginArgs``` parameter. **The full set of plugin arguments must be specified.**
```